format/avro/decoders/long.go

```diff
@@ -0,0 +1,37 @@
+package decoders
+
+import (
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+const intMask = byte(127)
+const intFlag = byte(128)
```

Comment:
Could use `0b111_1111` and `0xb1000_0000` to make it even more explicit? it the byte cast needed btw?